torch/csrc/jit/tensorexpr/function.cpp

```diff
@@ -111,6 +112,32 @@ Tensor* Compute(
   return new Tensor(buf, func, 0);
 }
 
+TORCH_API Tensor* Reduce(
+    const std::string& func_name,
+    const std::vector<DimArg>& dim_args,
+    const Reducer& reducer,
+    const std::vector<DimArg>& reduce_args) {
```

Comment:
what motivates multiple simultaneous reduction axes over nested reductions?

Reply:
It's simpler and doesn't require us to piece together the original reduction when lowering.